actions/templates/post/index.js

```diff
@@ -84,7 +89,8 @@ async function main(params) {
 
     const templateName = params[POST_PARAM_NAME];
     const githubRepoUrl = params[POST_PARAM_LINKS][POST_PARAM_LINKS_GITHUB];
-    const result = await findTemplateByName(templateName, params.TEMPLATE_REGISTRY_ORG, params.TEMPLATE_REGISTRY_REPOSITORY);
+
+    const result = await findTemplateByName(dbParams, templateName);
     if (null !== result) {
       return {
         'statusCode': 409
```

Comment:
We can remove checking the registry.json file below on line 100 since we're not using it anymore